### PR TITLE
Dump npm dependencies

### PR DESCRIPTION
**Pull request by davidhorak**
# Description
- Update NPM dependencies 

### Test Coverage
- [x] unit

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ module.exports = {
     'max-len': ['error', { code: 120 }],
     'no-dupe-class-members': 'off',
     'no-useless-constructor': 'off',
+    'no-undef': 'off',
```
Comment (rthreei) [inline .eslintrc.js:28]: Why is this necessary?

Comment (davidhorak) [reply .eslintrc.js:28]: https://github.com/briza-insurance/wegood/issues/43

**Review by rthreei — APPROVED** [review]
